radiation/python/radlw/lwrad_gt4py.py

```diff
@@ -0,0 +1,1800 @@
+    else:
+        locdict_gt4py[var] = create_storage_zeros(backend, shape_nlp1, DTYPE_FLT)
+
+# Initialize local vars for taumol
+for var in locvars_int:
+    if var == "ib":
+        locdict_gt4py[var] = create_storage_zeros(backend, shape_2D, DTYPE_INT)
+    elif var == "NGB":
+        locdict_gt4py[var] = create_storage_from_array(
+            np.tile(np.array(ngb)[None, None, :], (npts, 1, 1)),
+            backend,
+            shape_2D,
+            (DTYPE_INT, (ngptlw,)),
+            default_origin=(0, 0),
+        )
+    else:
+        locdict_gt4py[var] = create_storage_zeros(backend, shape_nlp1, DTYPE_INT)
+
+for var in locvars_flt:
+    if var == "taug":
+        locdict_gt4py[var] = create_storage_zeros(backend, shape_nlp1, type_ngptlw)
+    else:
+        locdict_gt4py[var] = create_storage_zeros(backend, shape_nlp1, DTYPE_FLT)
+
+# Initialize local vars for rtrnmc
+for var in locvars_rtrnmc:
+    if var[-3:] == "rad":
+        locdict_gt4py[var] = create_storage_zeros(backend, shape_nlp1, type_nbands)
+    elif var == "fnet" or var == "fnetc" or var == "rfdelp":
+        locdict_gt4py[var] = create_storage_zeros(backend, shape_nlp1, DTYPE_FLT)
+    elif var == "itgas" or var == "ittot":
+        locdict_gt4py[var] = create_storage_zeros(
+            backend, shape_nlp1, (np.int32, (ngptlw,))
+        )
+    elif var == "ib":
+        locdict_gt4py[var] = create_storage_zeros(backend, shape_2D, np.int32)
+    else:
+        locdict_gt4py[var] = create_storage_zeros(backend, shape_nlp1, type_ngptlw)
+
+locdict_gt4py["A0"] = create_storage_from_array(a0, backend, shape_nlp1, type_nbands)
+locdict_gt4py["A1"] = create_storage_from_array(a1, backend, shape_nlp1, type_nbands)
+locdict_gt4py["A2"] = create_storage_from_array(a2, backend, shape_nlp1, type_nbands)
+
+# Read in lookup table data for cldprop calculations
+ds = xr.open_dataset("../lookupdata/radlw_cldprlw_data.nc")
+
+absliq1 = ds["absliq1"].data
+absice0 = ds["absice0"].data
+absice1 = ds["absice1"].data
+absice2 = ds["absice2"].data
+absice3 = ds["absice3"].data
+
+lookup_dict = dict()
+lookup_dict["absliq1"] = create_storage_from_array(
+    absliq1, backend, (npts, 1, nlp1 + 1), (np.float64, (58, nbands))
+)
+lookup_dict["absice0"] = create_storage_from_array(
+    absice0, backend, (npts, 1, nlp1 + 1), (np.float64, (2,))
+)
+lookup_dict["absice1"] = create_storage_from_array(
+    absice1, backend, (npts, 1, nlp1 + 1), (np.float64, (2, 5))
+)
+lookup_dict["absice2"] = create_storage_from_array(
+    absice2, backend, (npts, 1, nlp1 + 1), (np.float64, (43, nbands))
+)
+lookup_dict["absice3"] = create_storage_from_array(
+    absice3, backend, (npts, 1, nlp1 + 1), (np.float64, (46, nbands))
+)
+lookup_dict["ipat"] = create_storage_from_array(
+    ipat, backend, (npts, 1, nlp1 + 1), (DTYPE_INT, (nbands,))
+)
+
+# Read in 2-D array of random numbers used in mcica_subcol, this will change
+# in the future once there is a solution for the RNG in python/gt4py
```

Comment:
this comment could be modified that this is just used for the test mode, when running the model a python random number generator is used. If it does ever end up in gt4py, we'll still want to use this netcdf file for testing.

Reply:
Yes, but right now that statement would not be true, as we don't yet have a random number generator in python in use